.env.example

```diff
@@ -6,7 +6,7 @@ export CLOUDAPI_URL=http://cloudapi.127.0.0.1.nip.io/tenant
 export GOVERNANCE_ACAPY_API_KEY=adminApiKey
 export GOVERNANCE_ACAPY_LABEL=Governance
 export GOVERNANCE_FASTAPI_ENDPOINT=http://cloudapi.127.0.0.1.nip.io/governance
-export LEDGER_REGISTRATION_URL=http://127.0.0.1:9000/register # http://ledger-browser.cloudapi.127.0.0.1.nip.io/register
+export LEDGER_REGISTRATION_URL=http://ledger-browser.cloudapi.127.0.0.1.nip.io/register
```

Comment:
Todo:
Make this `http://127.0.0.1:9000/register` if CPU is ARM based.
Else `http://ledger-browser.cloudapi.127.0.0.1.nip.io/register` for `x86_64`/`amd64`

Reply:
Alternatively, get Tilt to port-forward so this can remain `127.0.0.1:9000` regardless of CPU

Reply:
I went the port-forward route

Reply:
Doesn't look like `tilt ci` does port forwarding 😢 
https://github.com/tilt-dev/tilt/issues/5964